### PR TITLE
Create installation package for Snowflake

### PR DESCRIPTION
**Pull request by alex-graciano**
See https://github.com/CartoDB/carto-advanced-spatial-extension/pull/127

### CODE REVIEW

File: common/snowflake/Makefile [hunk 1]
```diff
@@ -22,6 +22,9 @@ NODE_MODULES_DEPS = $(COMMON_DIR)/node_modules
 REPLACEMENTS = 	-e 's!@@SF_PREFIX@@!$(SF_PREFIX)!g' -e 's!@@SF_DATABASE@@!$(SF_DATABASE)!g' -e 's!@@SF_SCHEMA@@!$(SF_SCHEMA)!g' -e 's!@@SF_SHARE@@!$(SF_SHARE)!g' -e '/@@SF_LIBRARY_CONTENT@@/ r $(SF_LIBRARY)' -e 's!@@SF_LIBRARY_CONTENT@@!!g'
 REPLACEMENTS += $(foreach f,$(notdir $(basename $(wildcard dist/*.js))),-e '/@@SF_LIBRARY_$(shell echo $f | tr a-z A-Z)@@/ r ./dist/$f.js' -e 's!@@SF_LIBRARY_$(shell echo $f | tr a-z A-Z)@@!!g' )
 
+REPLACEMENTS_PKG = -e 's!@@SF_PREFIX@@!!g' -e '/@@SF_LIBRARY_CONTENT@@/ r $(SF_LIBRARY)' -e 's!@@SF_LIBRARY_CONTENT@@!!g'
+REPLACEMENTS_PKG += $(foreach f,$(notdir $(basename $(wildcard dist/*.js))),-e '/@@SF_LIBRARY_$(shell echo $f | tr a-z A-Z)@@/ r ./dist/$f.js' -e 's!@@SF_LIBRARY_$(shell echo $f | tr a-z A-Z)@@!!g' )
```
Comment (vdelacruzb) [inline common/snowflake/Makefile:26]: This part is common with the top replacements, right? As the order doesn't really matter maybe you can define this code first as its common and then add it to `REPLACEMENTS`and `REPLACEMENTS_PKG`.

File: scripts/modulesort.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 // This script returns a list of the modules
-// ordered by checking their dependencies
+// ordered by checking their dependencies
```
Comment (vdelacruzb) [inline scripts/modulesort.js:4]: you are introducing here a space with no reason ^^

File: common/snowflake/Makefile [hunk 3]
```diff
@@ -111,6 +114,15 @@ ifeq ($(SF_SHARE_ENABLED), 1)
 	$(SED) $(REPLACEMENTS) $(SHARE_REMOVE_FILE) | $(SNOWSQL) -q "$(xargs)" 
 endif
 
+serialize-module:
```
Comment (vdelacruzb) [inline common/snowflake/Makefile:117]: this should be added to `.PHONY`I guess.

File: common/snowflake/Makefile (hunk 1 above)
Comment (alex-graciano) [reply common/snowflake/Makefile:26]: Right!

File: scripts/modulesort.js (hunk 2 above)
Comment (alex-graciano) [reply scripts/modulesort.js:4]: Yep, I needed to update just the submodule core and did this -.-. Updating in the next commit

File: common/snowflake/Makefile (hunk 3 above)
Comment (alex-graciano) [reply common/snowflake/Makefile:117]: You're right!

**Review by Jesus89 — APPROVED** [review]

**Pull request merged**